app/helpers/application_helper.rb

```diff
@@ -1214,7 +1215,7 @@ def start_page_allowed?(start_page)
   end
 
   def allowed_filter_db?(db)
-    return false if db.start_with?('Container') && !get_vmdb_config[:product][:containers]
+    return false if (db.nil? || db.start_with?('Container')) && !get_vmdb_config[:product][:containers]
     true
```

Comment:
> please, rewrite whole method to
> !db.to_s.start_with?('Container') || get_vmdb_config[:product][:containers]
> 
> if you are already touching it...

this method is going to be cleaned up anyway in another patch
cc @zakiva @simon3z 
(cleaning leftovers of a patch that was merged too early: https://github.com/ManageIQ/manageiq/pull/3684)
so I prefer not rewrite it as part of topology widget - it's not the primarily focus of this PR and should be treated separately.